scripts/zones/Beadeaux/npcs/GeFhu_Yagudoeye.lua

```diff
@@ -10,7 +10,7 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(124)
+    player:event(124)
```

Comment:
This isn't valid.  There is no `CLuaBaseEntity::event()` function

Reply:
Done ✅